lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -307,10 +307,15 @@ sub start
               host => "$bind_host",
               port => $self->{ports}{event_persister2},
            },
+	   "client_reader" => {
+              host => "$bind_host",
+              port => $self->{ports}{client_reader},
+           },
         },
 
         stream_writers => {
            events => $self->{redis_host} ne '' ? [ "event_persister1", "event_persister2" ] : "master",
+	   to_device => $self->{redis_host} ne '' ? [ "client_reader" ] : "master",
```

Comment:
There's no particular reason to choose `client_reader`, but I couldn't think of a better place and I'm not sure we want to add more workers at this point

Reply:
maybe stick those words in a comment

Reply:
Done :)